python/feathub/processors/flink/table_builder/tests/table_builder_test_utils.py

```diff
@@ -95,3 +98,31 @@ def _create_file_source(
             timestamp_format=timestamp_format,
             max_out_of_orderness=timedelta(minutes=1),
         )
+
+
+class MockTableDescriptor(TableDescriptor, ABC):
+    def __init__(
+        self,
+        name: str,
```

Comment:
It seems that none of the caller needs to specify name. Maybe it is simpler to just generate random name string here?